### ISSUE TITLE
SSH server takes forever to start, sesame.txt is unnecessary

### ISSUE THREAD

**Issue by ulfhermann** [majuss/easybox904#21]
As you are putting the dropbear start into rcS, it won't actually read the sesame.txt file we've dutifully put on a USB stick to be processed by the (disabled) dropbear start script. However, waiting 200s for the ssh server to start as the "sleep 200" there mandates is somewhat puzzling, especially to people who do the "quick" and easy method. Also, it's decisively bad style to just run an SSH server from rcS.

One way to do this in a nicer way is just fixing the dropbear start script by enabling the switch statement in the end, removing all the annoying "try to read sesame.txt" code and removing the "no password login" config options in the beginning (as we have no way to set those options before we log in for the first time, or do we?). The attached [dropbear.txt](https://github.com/majuss/easybox904/files/1844853/dropbear.txt) as /etc/init.d/dropbear (without .txt) works for me.

**Comment by ulfhermann** [comment]
And the code you suggest to put into rcS also starts a serial console that we don't use. The readme should at least tell the user about that.

**Comment by majuss** [comment]
Thanks again for the useful stuff, guide was updated and opened image now only contains your new dropbear service.

**Comment by majuss** [comment]
@ulfhermann with the latest vodafone firmware the check for the sesame.txt moved from the dropbear init file to "/usr/sbin/arc_dropbear" which is a compiled binary. I found no way to mitigate the check for the file... If you could take a look into it I would be very happy :)

**Comment by AnsgarRagentor** [comment]
Hi guys,

I´m def no pro in these things and I tried my best but  it seems I still can´t connect to the EB using the regular way proposed in the quick and easy guide. I created the sesame.txt via terminal under IOS using touch command and I patiently waited around 15 minutes being connected to the EB via WIFI... still with the same result: 

ssh: connect to host 192.168.2.1 port 22: Connection refused

It sounds like there has been another solution to this that doesn´t need the USB Stick to actually connect to the SSH Server. Does this still work with the newest firm-ware? Is there maybe something else I can try?

Thanks in advance!
